src/main/java/io/parsingdata/metal/data/ParseItemList.java

```diff
@@ -82,4 +85,19 @@ public String toString() {
         return isEmpty() ? "" : ">" + head + tail.toString();
     }
 
+    /**
+     * Return all {@link ParseGraph}s as list by iterating recursively.
+     * @return all {@link ParseGraph}s in <code>this</code> {@link ParseItemList}
+     */
+    public List<ParseGraph> asList() {
```

Comment:
Should probably be part of a `metal-utils`. @jvdb We should start that and put this and put some of @ccreeten's cool util methods in there as well.